util/config/run-in-venv-with-python-bindings.bash

```diff
@@ -17,4 +20,17 @@ if [ ! -d "$chpl_frontend_py_deps/chapel" ]; then
   exit 1
 fi
 
+if [ $("$python" "$CHPL_HOME/util/chplenv/chpl_sanitizers.py") == "address" ]; then
+  if [ "$PLATFORM" == "darwin" ]; then
+    echo "Cannot use chapel-py on Mac OS when address sanitization is enabled; please unset 'CHPL_SANITIZE', then rebuild Chapel and chapel-py"
+    exit 1
+  fi
+
+  if [ "$CC" == "clang" ]; then
+    export LD_PRELOAD=$($CC -print-file-name=libclang_rt.asan.so)
+  else
```

Comment:
note that you probably want some kind of substring check because `CHPL_HOST_CC` may be an absolute path

Reply:
good point, will add